release_notes/v2_vs_v3.adoc

```diff
@@ -16,6 +16,10 @@ Many of the same terms are used, and the same functions are performed, but the t
 
 This topic discusses these differences in detail, in an effort to help OpenShift users in the transition from OpenShift v2 to OpenShift v3.
 
+ifdef::openshift-dedicated[]
+OpenShift Dedicated uses the same code base as OpenShift Enterprise 3. OpenShift Dedicated is hosted on the public cloud and managed by Red Hat. An OpenShift Enterprise 3 subscription entitles you to host and manage the software on your infrastructure of choice.
+endif::[]
```

Comment:
Done, with some rewording.

I think with just the "using the same codebase as v3" sentence in there, it's probably not worth making the point at all. It says in the Overview and the Rel Notes that this is a thing. If anything, the important point to make would be that "This topic is relevant because Dedicated 3 uses the same codebase as Enterprise 3", which I think it's doing atm, but really, I'd be happy to take the notebox out and maybe even take this topic out of the Dedicated books.